sycl/include/CL/sycl/detail/queue_impl.hpp

```diff
@@ -19,154 +19,218 @@
-        m_PropList(PropList), m_HostQueue(m_Device->is_host()),
-        m_OpenCLInterop(!m_HostQueue) {
-    if (!m_HostQueue) {
-      m_CommandQueue = createQueue(Order);
+      : MDevice(Device), MContext(Context), MAsyncHandler(AsyncHandler),
+        MPropList(PropList), MHostQueue(MDevice->is_host()),
+        MOpenCLInterop(!MHostQueue) {
+    if (!MHostQueue) {
+      MCommandQueue = createQueue(Order);
     }
   }
 
-  queue_impl(cl_command_queue CLQueue, ContextImplPtr Context,
+  /// Constructs a SYCL queue from plugin interoperability handle.
+  ///
+  /// @param PiQueue is a raw PI queue handle.
+  /// @param Context is a pointer to context_impl.
+  /// @param AsyncHandler is a SYCL asynchronous exception handler.
+  queue_impl(RT::PiQueue PiQueue, ContextImplPtr Context,
              const async_handler &AsyncHandler)
-      : m_Context(Context), m_AsyncHandler(AsyncHandler),
-        m_HostQueue(false), m_OpenCLInterop(true) {
+      : MContext(Context), MAsyncHandler(AsyncHandler), MHostQueue(false),
+        MOpenCLInterop(true) {
 
-    m_CommandQueue = pi::cast<RT::PiQueue>(CLQueue);
+    MCommandQueue = pi::cast<RT::PiQueue>(PiQueue);
 
     RT::PiDevice Device = nullptr;
     // TODO catch an exception and put it to list of asynchronous exceptions
-    PI_CALL(piQueueGetInfo)(m_CommandQueue, PI_QUEUE_INFO_DEVICE,
-                            sizeof(Device), &Device, nullptr);
-    m_Device = std::make_shared<device_impl>(Device);
+    PI_CALL(piQueueGetInfo)(MCommandQueue, PI_QUEUE_INFO_DEVICE, sizeof(Device),
+                            &Device, nullptr);
+    MDevice = std::make_shared<device_impl>(Device);
 
     // TODO catch an exception and put it to list of asynchronous exceptions
-    PI_CALL(piQueueRetain)(m_CommandQueue);
+    PI_CALL(piQueueRetain)(MCommandQueue);
   }
 
   ~queue_impl() {
     throw_asynchronous();
-    if (m_OpenCLInterop) {
-      PI_CALL(piQueueRelease)(m_CommandQueue);
+    if (MOpenCLInterop) {
+      PI_CALL(piQueueRelease)(MCommandQueue);
     }
   }
 
+  /// @return an OpenCL interoperability queue handle.
   cl_command_queue get() {
-    if (m_OpenCLInterop) {
-      PI_CALL(piQueueRetain)(m_CommandQueue);
-      return pi::cast<cl_command_queue>(m_CommandQueue);
+    if (MOpenCLInterop) {
+      PI_CALL(piQueueRetain)(MCommandQueue);
+      return pi::cast<cl_command_queue>(MCommandQueue);
     }
     throw invalid_object_error(
         "This instance of queue doesn't support OpenCL interoperability");
   }
 
-  context get_context() const { return createSyclObjFromImpl<context>(m_Context); }
-
-  ContextImplPtr get_context_impl() const {
-    return m_Context;
+  /// @return an associated SYCL context.
+  context get_context() const {
+    return createSyclObjFromImpl<context>(MContext);
   }
 
-  device get_device() const { return createSyclObjFromImpl<device>(m_Device); }
+  /// @return a pointer to a context_impl.
+  ContextImplPtr get_context_impl() const { return MContext; }
```

Comment:
Shouldn't the function names be in format getContextImpl() ? That's what LLVM Coding standard is. 
I was under the impression that we only use this format at places where SYCL spec has defined APIs that way.
Eg: queue class has get_context() function. 
Please correct me here. 
@romanovvlad

Reply:
Sounds reasonable, thanks